src/css/utils/_typography.scss

```diff
@@ -42,6 +42,11 @@
   line-height: 1.75rem;
 }
 
+.text-lg-sm-lh {
```

Comment:
Is a different line height 100% necessary or can we go for changing it to the default line height for consistency? I guess it depends on whether this specific styling is used more than one time or not

Reply:
I adjusted with padding. Fixed in above-mentioned PR.